homeassistant/components/spokestack_tts/tts.py

```diff
@@ -0,0 +1,83 @@
+"""Support for the Spokestack speech service."""
+import logging
+
+from spokestack.tts.clients.spokestack import TextToSpeechClient, TTSError
+import voluptuous as vol
+
+from homeassistant.components.tts import CONF_LANG, PLATFORM_SCHEMA, Provider
+import homeassistant.helpers.config_validation as cv
+
+from .const import (
+    CONF_KEY_ID,
+    CONF_MODE,
+    CONF_SECRET_KEY,
+    CONF_VOICE,
+    DEFAULT_LANG,
+    DEFAULT_MODE,
+    DEFAULT_VOICE,
+    SUPPORTED_LANGUAGES,
+    SUPPORTED_MODES,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
```

Comment:
This should be converted to a config entry.

Reply:
For this one, you mean to add the API keys via [config_flow](https://developers.home-assistant.io/docs/config_entries_config_flow_handler#defining-steps)? just wanted to verify I was adding it correctly in the updates.

Reply:
Yes.